book/src/getting_started.md

```diff
@@ -35,3 +35,27 @@ fn main() {
 ```
 
 You should now be able to compile and run this code.
+
+## Using integrations
+
+To use one of the [integrations](./integrations.md), with axum as an example:
+
+First, add this to your `Cargo.toml` instead:
+
+```toml
+# in section [dependencies]
+askama_axum = "0.4.0"
+
+```
+
+Then, import from askama_axum instead of askama:
+
+```rust
+//use askama::Template;
```

Comment:
Forgot this maybe?

Reply:
This is a visualization for "instead" ;) if you don't feel like it I'll remove it.

Reply:
it's a bit strange. I'll let another reviewer give the last word here.